### PR TITLE
Include role-service

### PR DESCRIPTION
**Pull request by tamirkamara**
## Description
Update deployment so that services will work.

### Changes
- Add Role service
- Remove private links for now as they cause problems (there's a separate task to fix that)
- Align port numbers
- Remove duplicate script by using a tfvars file.

### Checklist:

- [ ] Branch name follows convention (feature/arts-###-short-name OR fix/arts-###-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

### CODE REVIEW

File: tamir2/definition.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "tamir2",
```
Comment (eladiw) [inline tamir2/definition.json]: need to delete this file

File: tamir2/services/application.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+management:
```
Comment (eladiw) [inline tamir2/services/application.yml]: delete the entire dir

File: terraform/trial_rg/modules/kv/main.tf [hunk 3]
```diff
@@ -15,19 +15,21 @@ resource "azurerm_key_vault" "trial_keyvault" {
   }
 }
 
+# TODO: currect this is uncomment
```
Comment (eladiw) [inline terraform/trial_rg/modules/kv/main.tf]: minor. sentence with typos

File: terraform/trial_rg/modules/kv/main.tf [hunk 4]
```diff
@@ -15,19 +15,21 @@ resource "azurerm_key_vault" "trial_keyvault" {
   }
 }
 
+# TODO: currect this and uncomment
```
Comment (eladiw) [inline terraform/trial_rg/modules/kv/main.tf]:
```suggestion
# TODO: correct this and uncomment
```

**Review by eladiw — DISMISSED** [review]

**Review by shajidissa — APPROVED** [review]

**Review by eladiw — APPROVED** [review]

**Pull request merged**